README.md

```diff
@@ -0,0 +1,50 @@
+Mender Update Modules
+=====================
+
+Mender is an open source over-the-air (OTA) software updater for embedded Linux devices. Mender comprises a client running at the embedded device, as well as a server that manages deployments across many devices.
+
+This repository contains community supported Update Modules. An Update Module is an extension to the Mender client for supporting a new type of software update, such as a package manager, container, bootloader or even updates of nearby microcontrollers. An Update Module can be tailored to a specific device or environment (e.g. update a proprietary bootloader), or be more general-purpose (e.g. install a set of .deb packages.).
+
+There are a couple of general-purpose Update Modules bundled together with the [Mender client source code](https://github.com/mendersoftware/mender/tree/master/support/modules).
+
+![Mender logo](https://mender.io/user/pages/resources/06.digital-assets/mender.io.png)
+
+## Getting started
+
+To start using Mender, we recommend that you begin with the Getting started
+section in [the Mender documentation](https://docs.mender.io/).
+
+To start using Mender Update Modules, we recommend that you begin with the Update Modules - Introduction
+section in [the Mender documentation](https://docs.mender.io/devices/update-modules).
+
+You can find detailed information about available Update Modules on [Mender Hub](https://hub.mender.io/c/update-modules).
+
+## Contributing
+
+We welcome and ask for your contribution. If you would like to contribute to Mender, please read our guide on how to best get started [contributing code or
+documentation](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md).
+
+## License
+
+Mender is licensed under the Apache License, Version 2.0. See
+[LICENSE](https://github.com/mendersoftware/mender-update-modules/blob/master/LICENSE) for the
+full license text.
```

Comment:
I think this is not appropriate here, because this is not the Mender client. And in fact the license of the client has no impact on this repository at all.

It should still apply for update modules we accept though, but maybe we should phrase it a bit differently:

```
All files in this repository is licensed under the Apache License, Version 2.0, unless indicated otherwise.
[LICENSE](https://github.com/mendersoftware/mender-update-modules/blob/master/LICENSE) for the
full license text.
```